src/App.tsx

```diff
@@ -10,15 +9,16 @@ import Login from './pages/Login';
 import Signup from './pages/Signup';
 import Navbar from './components/Navbar';
 
+import ProfilePage from './pages/ProfilePage';
+
 function App() {
   return (
     <main>
-      <TempNavbar />
```

Comment:
By not having the temp nav bar, I have found no easy way to navigate to yours or Sepand's pages

Reply:
I would suggest keeping in the tempnavbar, at least for demonstration purposes in this sprint